src/mlagility/api/devices.py

```diff
@@ -386,6 +386,7 @@ def execute_trt_remotely(
     _ip, username = configure_remote(device)
```

Comment:
increment patch version number of mlagility so we can release to PyPI when this merges :)